### ISSUE TITLE
VPAT - Header 1 / Header Hierarchy

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#2309]
Web Accessibility Evaluation Tool ("WAVE") shows an **alert** that the header hierarchy is incorrect.

WAVE explains:
>Headings facilitate page navigation for users of many assistive technologies. They also provide semantic and visual meaning and structure to the document. A first level heading (\<h1\>) should be present on nearly all pages. It should contain the most important heading on the page (generally the document title).

Analysis
- The global header (first view that is rendered) begins with an \<h2\> tag to display a hello message.
-  Most page views do not have a \<h1\> tag.  This generates an alert: "A page does not have a first level heading".
- Those views that do have an \<h1\> tag have this alert, instead: "A heading level is skipped."
  - Lichen Plagiarism view

Recommendation
- Change the "hello" message from \<h2\> to \<h1\>.  This will make the "hello" message larger.  Therefore, use CSS to restore message to original text size.
- Those page views that do have an \<h1\> tag should be downgraded to a lower level, and then use CSS to restore text size.  Otherwise, the "A heading level is skipped." alert will persist.

**Comment by pbailie** [comment]
The full text from WAVE is as follows (in regards to views that have no \<h1\> tag):
>**What It Means**
A page does not have a first level heading.
**Why It Matters**
Headings facilitate page navigation for users of many assistive technologies. They also provide semantic and visual meaning and structure to the document. A first level heading (\<h1\>) should be present on nearly all pages. It should contain the most important heading on the page (generally the document title).
**How to Fix It**
If the page presents a main heading, place it within an \<h1\> element. Add other sub-headings as necessary.
**The Algorithm... in English**
A page does not have an \<h1\> element.
**Standards and Guidelines**
[Section 508 (o)](http://webaim.org/standards/508/checklist#standardo)
[1.3.1 Info and Relationships (Level A)](http://webaim.org/standards/wcag/checklist#sc1.3.1)
[2.4.2 Page Titled (Level A)](http://webaim.org/standards/wcag/checklist#sc2.4.2)
[2.4.6 Headings and Labels (Level AA)](http://webaim.org/standards/wcag/checklist#sc2.4.6)

**Comment by MasterOdin** [comment]
We shouldn't elevate the "Hello, <display_name>" to h1 as that's not really the "document title".

I think to best follow this particular warning, we'll want to rewrite the "Hello" and Breadcrumb lines to not use `<h>` tags (instead just `<div>` with CSS styling) and then make sure to place a `<h1>` tag within the box element (or above it) on each page.

**Comment by mcglow2** [comment]
@pbailie the current code base appears to use <h> tags much better.  The main outstanding item in this issue is the use of the <h2> tag in the header for "Submitty" and the bread crumbs.  Per the recommendation above can you change the H2 to a div and use css to maintain the exact same visual look.
